db/orders.js

```diff
@@ -1,44 +1,42 @@
 const client = require("./client")
 
-async function addToOrderItems({
+async function createOrder({
     userId,
-    puppyId,
     date,
     status,
 }) {
     try{
-        const { rows: [order_item] } = await client.query(`
-            INSERT INTO order_items("userId", "puppyId", date, status)
-            VALUES ($1, $2, $3, $4)
-            ON CONFLICT ("userId", "puppyId") DO NOTHING
+        const { rows: [order] } = await client.query(`
+            INSERT INTO orders("userId", date, status)
+            VALUES ($1, $2, $3)
             RETURNING *;
-        `, [userId, puppyId, date, status])
+        `, [userId, date, status])
 
-    return order_item;
+    return order;
     } catch (error) {
         console.error(error)
     }
 }
 
-async function getOrderItemById(id) {
+async function getOrderById(id) {
 try{
-        const { rows: [ order_item ] } = await client.query(`
+        const { rows: [ order ] } = await client.query(`
             SELECT *
-            FROM order_items
+            FROM orders
             WHERE id=$1;
         `, [id])
 
-        return order_item;
+        return order;
     } catch (error) {
         console.error(error)
     }
 }
 
-async function getOrderItemByUser({ id }) {
+async function getOrderByUser({ id }) {
```

Comment:
This function does what we want (good job), but I think getOrdersByUser would be more accurate since it is getting all the orders from the table with the matching userId.